WalletWasabi.Gui/Tabs/EncryptionManager/EncryptMessageView.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui" Design.Width="600">
+  <controls:GroupBox Title="{Binding Title}" BorderThickness="0" Classes="docTabContainer">
+    <Grid Classes="content" Margin="0 40 0 0">
+      <Grid.RowDefinitions>
+        <RowDefinition Height="*" />
+        <RowDefinition Height="Auto" />
+        <RowDefinition Height="*" />
+        <RowDefinition Height="Auto" />
+      </Grid.RowDefinitions>
+      <controls:ExtendedTextBox Grid.Row="0" Text="{Binding PlainMessage}" Watermark="Message" UseFloatingWatermark="True" TextWrapping="Wrap" AcceptsReturn="True" />
+      <controls:ExtendedTextBox Grid.Row="1" Text="{Binding PublicKey}" Watermark="Public key of recipient" UseFloatingWatermark="True" />
```

Comment:
The `"Public key of recipient"` is formally called `Encryption key` and Electrum calls it simply `Public key`. I personally prefer `Encryption key` or `Public key`.

Reply:
good! fixed!